cli/flags.rs

```diff
@@ -658,6 +665,14 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
         .help("Check if the source files are formatted.")
         .takes_value(false),
     )
+    .arg(
+      Arg::with_name("ignore")
+        .long("ignore")
+        .takes_value(true)
+        .use_delimiter(true)
+        .require_equals(true)
+        .help("Ignore formatting particular source files."),
+    )
```

Comment:
I think if you make this dependent on the unstable flag `.requires("unstable")`, we can land this before 1.3.0. It should be reverted when the time comes.

Reply:
Sure. Will do.

Reply:
I don't think it should be dependent on the `--unstable` flag. Nothing much unstable in it anyways. It just is an extra step between fmt that filters through the vec.

Reply:
It's an interface change - so it needs to wait for 1.3.0 (August 13). If it is marked as unstable we can land it now.

Reply:
Alrighty! Will do.